### DISCUSSION TITLE
Handle restoring the image when validation failed

### DISCUSSION THREAD

**Discussion by thanhhungtn1** [rahulhaque/laravel-filepond#65] [Q&A] [answered]
Hi I have read the Readme and the codes, but not sure how I can restore the image when submit the form when the validation is failed.
I see no restore route in the route list so kinda confused how to restore the image when form reload. 
Thank you so much for helping!

**Comment by rahulhaque** [comment, marked as answer]
@thanhhungtn1 you have to write the restore endpoint logic yourself. This will vary based on the use cases which is why I don't have this endpoint included in my package. You can have a look here for implementation instructions - https://pqina.nl/filepond/docs/api/server/#restore

**Reply by thanhhungtn1** [reply]
Thank you for quick respond man. Appreciate it.
